platform/ui/src/components/userPreferencesModal/__docs__/hotkeyDefaults.js

```diff
@@ -55,16 +55,11 @@ export default {
     column: 1,
   },
   toggleCinePlay: { label: 'Play/Pause Cine', keys: ['SPACE'], column: 1 },
-  toggleCineDialog: {
+  F: {
```

Comment:
Not sure I follow. These are usually `commandNames` + `keys` combinations

I don't think we have a command named `F`, but that does sound like it could be the key(s) you press to fire the command.

Reply:
That was a huge gigantic typo, sorry, ill update this code.

Reply:
Done! thx.